libqtile/command/interface.py

```diff
@@ -148,10 +148,13 @@ def execute(self, call: CommandGraphCall, args: Tuple, kwargs: Dict) -> Any:
             The keyword arguments to pass into the command graph call.
         """
         obj = self._command_object.select(call.selectors)
-
+        cmd = None
         try:
             cmd = obj.command(call.name)
         except SelectError:
+            pass
+
+        if cmd is None:
```

Comment:
As above, you could do away with this if we had a `no_op` function returned.

Reply:
Would that fix it? Is the problem with this code (before your addition) not that `cmd` might never be set if a `SelectError` is raised?

Reply:
You're right. That wouldn't fix the SelectError issue so maybe it's better as I've got it here.